spec/models/about_spec.rb

```diff
@@ -6,6 +6,68 @@
     include_examples 'stats cacheable'
   end
 
+  describe "#stats" do
+    after do
+      About.clear_plugin_stat_groups
+    end
+
+    it "adds plugin stats to the output" do
+      stats = { last_day: 1, "7_days" => 10, "30_days" => 100, count: 1000 }
+      About.add_plugin_stat_group("some_group", show_in_ui: true) do
+        stats
+      end
+      expect(described_class.new.stats.with_indifferent_access).to match(
+        hash_including(
+          some_group_last_day: 1,
+          some_group_7_days: 10,
+          some_group_30_days: 100,
+          some_group_count: 1000,
+        )
+      )
+    end
+
+    it "does not add plugin stats to the output if they are missing one of the required keys" do
+      stats = { "7_days" => 10, "30_days" => 100, count: 1000 }
+      About.add_plugin_stat_group("some_group", show_in_ui: true) do
+        stats
+      end
+      expect(described_class.new.stats).not_to match(
+        hash_including(
+          some_group_last_day: 1,
+          some_group_7_days: 10,
+          some_group_30_days: 100,
+          some_group_count: 1000,
+        )
+      )
+    end
+
+    it "does not error if any of the plugin stat blocks throw an error and still adds the non-errored stats to output" do
+      stats = { last_day: 1, "7_days" => 10, "30_days" => 100, count: 1000 }
+      About.add_plugin_stat_group("some_group", show_in_ui: true) do
+        stats
+      end
+      About.add_plugin_stat_group("other_group", show_in_ui: true) do
+        raise StandardError
+      end
+      expect(described_class.new.stats.with_indifferent_access).to match(
+        hash_including(
+          some_group_last_day: 1,
+          some_group_7_days: 10,
+          some_group_30_days: 100,
+          some_group_count: 1000,
+        )
+      )
+      expect(described_class.new.stats.with_indifferent_access).not_to match(
+        hash_including(
+          other_group_last_day: 1,
+          other_group_7_days: 10,
+          other_group_30_days: 100,
+          other_group_count: 1000,
+        )
+      )
+    end
```

Comment:
I don't think this assertion is actually useful because both the keys and counts will not exist in the hash even if an error is not raised. I think an assertion to ensure `About.new.stats` does not error is sufficient.